config/power_virtual_servers_regions.yml

```
@@ -0,0 +1,40 @@
+us-south:
+  :name: us-south
+  :description: Dallas, TX United States (deprecated)
+  :hostname: us-south.power-iaas.cloud.ibm.com
```

Need to add a "Dallas 12" entry to parse services with `dal12` location:
```suggestion
  :hostname: us-south.power-iaas.cloud.ibm.com
dal:
  :name: dal
  :description: Dallas, TX United States
  :hostname: dal.power-iaas.cloud.ibm.com
```